tests/helpers/module-for-acceptance.js

```diff
@@ -6,6 +6,9 @@ import destroyApp from '../helpers/destroy-app';
 export default function(name, options = {}) {
   module(name, {
     beforeEach() {
+      // clear all user information in local storage so that it doesn't persist between tests
+      window.localStorage.clear();
```

Comment:
Gatekeeper uses local storage which causes bad state to persist between tests. Clear it before and after (see below) each test

Reply:
Nice, so this was the problem.